README.md

```diff
@@ -37,15 +45,15 @@ which means Ruby ≤ 2.4 is unmaintained and considered insecure.
 
 The action works on these [GitHub-hosted runners](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/virtual-environments-for-github-hosted-runners) virtual environments. Virtual environments not listed below are unsupported.
 
-| Operating System | Recommended | Other Supported Versions |
-| ----------- | -------- | -------- |
-| Ubuntu  | `ubuntu-latest`  (= `ubuntu-20.04`) | `ubuntu-18.04`, `ubuntu-22.04` |
-| macOS   | `macos-latest`   (= `macos-11`)  | `macos-10.15`, `macos-12` (beta) |
-| Windows | `windows-latest` (= `windows-2022`) | `windows-2019` |
+| Operating System | Recommended                         | Other Supported Versions         |
+| ---------------- | ----------------------------------- | -------------------------------- |
+| Ubuntu           | `ubuntu-latest`  (= `ubuntu-20.04`) | `ubuntu-18.04`, `ubuntu-22.04`   |
+| macOS            | `macos-latest`   (= `macos-11`)     | `macos-10.15`, `macos-12` (beta) |
+| Windows          | `windows-latest` (= `windows-2022`) | `windows-2019`                   |
```

Comment:
I don't want to manually maintain alignment here, it means lots of extra lines to change.

Reply:
So don't.  It's a lot more readable than the current.